lib/features.js

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+module.exports = config => {
+  const lastAccessTimeUpdates = config.lastAccessTimeUpdates
+  const signinConfirmation = config.signinConfirmation
+
+  return {
+    isSampledUser: isSampledUser,
+
+    isLastAccessTimeEnabled (uid, email) {
+      return lastAccessTimeUpdates.enabled && (
+        isSampledUser(uid, lastAccessTimeUpdates.sampleRate) ||
+        new RegExp(lastAccessTimeUpdates.enabledEmailAddresses).test(email)
+      )
+    },
+
+    isSigninConfirmationEnabled (uid, email, request) {
```

Comment:
I'm ok with the name change, however at a glance `isSigninConfirmationEnabled` was read by me as "is this feature enabled for the system" not necessarily by user. Adding a jsdoc to file might help clear up some of the confusion.